src/PrestaShopBundle/Resources/config/services/adapter/supplier.yml

```diff
@@ -0,0 +1,59 @@
+services:
+  _defaults:
+    public: true
+
+  prestashop.adapter.supplier.provider.supplier_logo:
+    class: 'PrestaShop\PrestaShop\Adapter\Supplier\SupplierLogoThumbnailProvider'
+    arguments:
+      - '@prestashop.core.image.parser.image_tag_source_parser'
+      - "@=service('prestashop.adapter.legacy.context').getContext().shop.id"
+
+  prestashop.adapter.supplier.provider.supplier_address:
+    class: 'PrestaShop\PrestaShop\Adapter\Supplier\SupplierAddressProvider'
+
+  prestashop.adapter.supplier.supplier_order_validator:
+    class: 'PrestaShop\PrestaShop\Adapter\Supplier\SupplierOrderValidator'
+
+# Command handlers
+
+  prestashop.adapter.supplier.command_handler.abstract_delete_supplier_handler:
+    class: PrestaShop\PrestaShop\Adapter\Supplier\CommandHandler\AbstractDeleteSupplierHandler
```

Comment:
I think you do not need to declare this service (and it cannot be used as it is abstract)

Reply:
I'm using it in two command handlers like this defining this service as a parent:

``` yml
prestashop.adapter.supplier.command_handler.delete_supplier_handler:
    class: PrestaShop\PrestaShop\Adapter\Supplier\CommandHandler\DeleteSupplierHandler
    parent: prestashop.adapter.supplier.command_handler.abstract_delete_supplier_handler
    tags:
      - name: tactician.handler
        command: PrestaShop\PrestaShop\Core\Domain\Supplier\Command\DeleteSupplierCommand
    public: true
```

however, since this abstraction is not supposed to be used directly I'll add an argument:

``` yml
public: false
```

Thanks for notice :+1: